packages/search-ui/src/requestAdapters.js

```diff
@@ -0,0 +1,34 @@
+function removeName(v) {
+  if (v && v.name) {
+    // eslint-disable-next-line
+    const { name, ...rest } = v;
+    return {
+      ...rest
+    };
+  }
+
+  return v;
+}
+
+function rollup(f) {
+  // TODO Move this to the app search connector and just say "app search connector"
```

Comment:
Just flagging this TODO in here

Reply:
@byronhulcher Thanks, and thanks for reviewing this. I have another PR queued which cleans this up.